scram/route_manager/models.py

```diff
@@ -97,7 +99,7 @@ class Entry(models.Model):
     history = HistoricalRecords()
     when = models.DateTimeField(auto_now_add=True)
     who = models.CharField("Username", default="Unknown", max_length=30)
-    expiration = models.DateTimeField(default="9999-12-31 00:00")
+    expiration = models.DateTimeField(default=datetime(9999, 12, 31, 0, 0, tzinfo=timezone.utc))
```

Comment:
Looks like you should remove `from django.utils import timezone` and change this to `tzinfo=datetime.timezone.utc`

Reply:
Updated to change to that.

Reply:
I'm not seeing this update?

Reply:
Helps if I actually push..